.travis.yml

```diff
@@ -1,13 +1,14 @@
 script:
   - mkdir build
   - cd build
-  - cmake ${CMAKEFLAGS} -DCMAKE_CXX_COMPILER=${CXX} -DCMAKE_C_COMPILER=${CC} ..
+  - cmake ${CMAKEFLAGS} -DCMAKE_CXX_COMPILER=${CXX} ..
   - make
   - make test
   - make package
 
 sudo: required
 dist: trusty
+language: cpp
```

Comment:
This was a workaround as otherwise the environment would override the CXX definition, causing the builds to all use GCC 4.9.*. Does that now work properly?